python/hail/expr/functions.py

```diff
@@ -1065,6 +1065,40 @@ def or_missing(predicate, value):
     predicate = to_expr(predicate)
     return _func("orMissing", predicate._type, predicate, value)
 
+@typecheck(x=expr_int32, n=expr_int32, p=expr_numeric,
+           alternative=enumeration("two.sided", "greater", "less"))
+@args_to_expr
+def binom_test(x, n, p, alternative):
+    """Performs an exact binomial test on `x` with `n` trials at probability `p`.
+
+    Examples
+    --------
+    .. doctest::
+
+        >>> eval_expr(f.binom_test(5, 10, 0.5, 'less'))
+        0.6230468749999999
+
+    Returns the p-value from the `exact binomial test
```

Comment:
Instead of this, use (with appropriate backtick-ing):
With alternative ``lesser``, the p-value is the probability of at most `x` successes, i.e. the cumulative probability at `x` of the distribution Binom(n, p). With ``greater``, the p-value is the probably of at least `x` successes. With ``two.sided``, the p-value is the total probability of all outcomes with probability at most that of `x`.